packages/manager/src/views/WebPropertyDetailPage/components/SSR/ContainerizedDeployment.tsx

```diff
@@ -286,228 +296,258 @@ export const ContainerizedDeployment = () => {
-            perPage={perPage}
-            page={page}
-            perPageOptions={perPageOptions}
-            variant={PaginationVariant.top}
-            onSetPage={onPageSet}
-            onPerPageSelect={onPerPageSelect}
-          />
-        </SplitItem>
-      </Split>
-      <TableComposable aria-label="spa-property-list" variant="compact" isStriped>
-        <Thead noWrap>
-          <Tr>
-            <Th modifier="wrap" width={15}>
-              SPA Name
-            </Th>
-            <Th modifier="wrap" width={15}>
-              Environments
-            </Th>
-            <Th modifier="wrap" width={15}>
-              Reference
-            </Th>
-            <Th modifier="wrap" width={15}>
-              Path
-            </Th>
-            <Th modifier="wrap" width={15}>
-              HealthCheck Path
-            </Th>
-            <Th modifier="wrap" width={25}>
-              Router URL
-            </Th>
-            <Th
-              modifier="wrap"
-              width={20}
-              style={{ justifyContent: 'space-evenly', display: 'grid' }}
-            >
-              Actions
-            </Th>
-          </Tr>
-        </Thead>
-        {(spaProperties.isLoading && webProperties.isLoading) ||
-        (spaProperties.isLoading && isSpaPropertyListEmpty) ? (
-          <TableRowSkeleton rows={3} columns={7} />
-        ) : (
-          <Tbody>
-            {paginatedData?.map((val) => (
-              <Tr key={val.name}>
-                <Td style={{ maxWidth: '15ch', wordWrap: 'break-word' }}>
-                  {`${val?.name.slice(0, URL_LENGTH_LIMIT)} ${
-                    val?.name.length > URL_LENGTH_LIMIT ? '...' : ''
-                  }`}
-                </Td>
-                <Td style={{ maxWidth: '15ch', wordWrap: 'break-word' }}>
-                  <Label
-                    key={val.env}
-                    icon={val.isGit && <GithubIcon />}
-                    color={val.isContainerized || val.isGit ? 'cyan' : 'gold'}
-                    isCompact
-                    style={{ marginRight: '8px' }}
-                  >
-                    {val.env}
-                  </Label>
-                </Td>
-                <Td style={{ maxWidth: '15ch', wordWrap: 'break-word' }}>{`${val?.ref.slice(
-                  0,
-                  URL_LENGTH_LIMIT
-                )} ${val?.ref.length > URL_LENGTH_LIMIT ? '...' : ''}`}</Td>
-                <Td style={{ maxWidth: '15ch', wordWrap: 'break-word' }}>{`${val?.path.slice(
-                  0,
-                  URL_LENGTH_LIMIT
-                )} ${val?.path.length > URL_LENGTH_LIMIT ? '...' : ''}`}</Td>
-                <Td style={{ maxWidth: '15ch', wordWrap: 'break-word' }}>
-                  {`${val?.healthCheckPath.slice(0, URL_LENGTH_LIMIT)} ${
-                    val?.healthCheckPath.length > URL_LENGTH_LIMIT ? '...' : ''
-                  }`}
-                </Td>
+        <SplitItem isFilled>
```

Comment:
Can you add a condition when there is no deployment then env filter should not be visible , so when empty state is there then env filter should not be visible

Reply:
Okay

Reply:
done